tests/Integration/Momento.Sdk.Tests/SimpleCacheControlTest.cs

```diff
@@ -74,15 +74,15 @@ public async Task ListCachesAsync_OneCache_HappyPath()
 
         // Test cache exists
         ListCachesResponse result = await client.ListCachesAsync();
-        Assert.True(result is ListCachesResponse.Success);
+        Assert.True(result is ListCachesResponse.Success, $"Unexpected response: {result}");
```

Comment:
Do we need `Unexpected response` on Success?

Reply:
@poppoerika These messages are printed when the assertion is not satisfied, so the result being printed in the message is **not** a success.

Reply:
👍